### PR TITLE
DAOS-8488 doc: tiered storage config

### PR DESCRIPTION
**Pull request by tanabarr**
Update admin guide instructions to take account of tiered storage config.

Signed-off-by: Tom Nabarro <tom.nabarro@intel.com>

### CODE REVIEW

File: docs/admin/deployment.md [hunk 1]
```diff
@@ -562,29 +634,66 @@ PCI:0000:da:00.0 Model:INTEL SSDPED1K750GA  FW:E2010435 Socket:1 Capacity:750 GB
...
-The next step consists of adjusting in the server configuration the storage
-devices that should be used by DAOS. The `servers` section of the yaml is a
-list specifying details for each DAOS I/O instance to be started on the host
-(currently a maximum of 2 per host is imposed).
+#### Selection
+
+The next step consists of specifying the devices that should be used by DAOS
+in the server configuration YAML file.
+The `engines` section of the config is a list with details for each DAOS engine
+to be started on the host (currently a maximum of 2 per host is imposed).
```
Comment (mjmac) [inline docs/admin/deployment.md:668]: There hasn't been a limit on the number of storage engines for quite a while. Probably a good idea to fix that now.

File: docs/admin/deployment.md [hunk 2]
```diff
@@ -633,11 +745,14 @@ engines:
...
-  scm_mount: /mnt/daos
-  scm_class: dcpm
-  scm_list: [/dev/pmem1]       # <----- updated
-  bdev_class: nvme
-  bdev_list: ["0000:da:00.0"]  # <----- updated
+  storage:
+  -
+    class: dcpm               # type of first storage tier (SCM)
+    scm_list: [/dev/pmem1]    # <----- updated
+    scm_mount: /mnt/daos      # where to mount SCM
```
Comment (mjmac) [inline docs/admin/deployment.md:752]: Should probably show that each engine config has a unique SCM mountpoint as well, to avoid confusion.

**Review by mjmac — COMMENTED** [review]
Looks good, just a couple of nits.

**Review by cdurf1 — APPROVED** [review]

**Review by mjmac — DISMISSED** [review]

**Review by cdurf1 — DISMISSED** [review]

File: docs/admin/administration.md [hunk 3]
```diff
@@ -344,6 +343,30 @@ boro-11
         Device Reliability: OK
         Read Only: OK
         Volatile Memory Backup: OK
+      Intel Vendor SMART Attributes:
+        Program Fail Count:
+           Normalized(%%):100
```
Comment (sydneyvanda) [inline docs/admin/administration.md:348]: the "%%" output is a mistake - I thought I fixed this already but clearly not. I will file a ticket to address

**Review by sydneyvanda — APPROVED** [review]
@tanabarr Thanks for adding the SMART stats output in the docs!

**Review by mjmac — APPROVED** [review]

**Review by cdurf1 — APPROVED** [review]

**Review by sydneyvanda — APPROVED** [review]

**Pull request merged**